src/workflow/ArcanistDiffWorkflow.php

```diff
@@ -2804,11 +2807,20 @@ private function pushChangesToStagingArea($id) {
       $ref_list[] = $ref['commit'].':'.$ref['ref'];
     }
 
-    $err = phutil_passthru(
-      'git push %Ls -- %s %Ls',
-      $push_flags,
-      $staging_uri,
-      $ref_list);
+    // phutil_passthru wraps the given string with single quotes
+    // therefore, "GIT_TRACE=1" is not passed through %s in string formatting
+    if ($this->getArgument('trace-git-push'))
+      $err = phutil_passthru(
+        'GIT_TRACE=1 git push %Ls -- %s %Ls',
```

Comment:
--trace is used in parsing level, and not clear how to get it at workflow level. any ideas @Pawka ?